src/app/src/components/EmbeddedMapTextSearchLabelConfig.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect } from 'react';
+import Typography from '@material-ui/core/Typography';
+import { withStyles } from '@material-ui/core/styles';
+import ControlledTextInput from './ControlledTextInput';
+import { DEFAULT_SEARCH_TEXT } from '../util/constants';
+
+const styles = {
+    subsectionHeader: {
+        color: 'rgb(0, 0, 0)',
+        fontSize: '15px',
+        fontWeight: 'bold',
+        margin: '10px 0',
+    },
+};
+
+function EmbeddedMapTextSearchLabelConfig({
+    textSearchLabel,
+    setTextSearchLabel,
+    anyFieldSearchable,
+}) {
+    const setTextToDefault = () => {
+        if (!textSearchLabel) {
+            setTextSearchLabel(DEFAULT_SEARCH_TEXT);
+        }
+    };
+
+    useEffect(() => {
+        if (!anyFieldSearchable) {
+            setTextSearchLabel(DEFAULT_SEARCH_TEXT);
+        }
+    }, [anyFieldSearchable]);
+
+    return (
+        <>
+            <Typography style={styles.subsectionHeader}>
+                Custom search text label
+            </Typography>
+            <Typography>
+                Set the text to display on your side bar when fields are marked
+                as searchable. Default is &quot;{DEFAULT_SEARCH_TEXT}
+                &quot;
```

Comment:
"If you've made any of your custom fields searchable, you can write your own custom label for the search box, so users know what they can search for."

Reply:
It would be awesome if instead of "Default is…" you could put a link that says "Reset to default". It would only be visible if the field isn't empty. Onclick, it would clear the field (and disappear itself).